docs/data/material/getting-started/templates/marketing-page/components/LogoCollection.tsx

```diff
@@ -30,8 +30,19 @@ const logoStyle = {
 };
 
 export default function LogoCollection() {
-  const theme = useTheme();
-  const logos = theme.palette.mode === 'light' ? darkLogos : whiteLogos;
```

Comment:
Why this didn't work?

Reply:
There are 2 issues with this code

1. theme.palatte.mode isn't returning updated `mode` value when user changes theme. `useColorScheme` is returning correct mode, hence i used that
2. `system` mode is not properly handled here.